_integration-tests/tests/pgx/pgx.go

```diff
@@ -45,12 +45,18 @@ func (tc *TestCase) Setup(t *testing.T) {
 		),
 	)
 	utils.AssertTestContainersError(t, err)
+	utils.RegisterContainerCleanup(t, tc.container)
 
 	dbURL, err := tc.container.ConnectionString(ctx, "sslmode=disable")
 	require.NoError(t, err)
 
 	tc.conn, err = pgx.Connect(ctx, dbURL)
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+		defer cancel()
+		assert.NoError(t, tc.conn.Close(ctx))
```

Comment:
Do we use the require or assert package ? Can we normalize that ?

Reply:
we can use both:
- `require` is an assertion followed by `t.FailNow`: I typically use this when this check is required for the test to continue (e.g. the rest of the test cannot even run if the container fails to start, or if I'm gonna be looking for specific indexes in a slice, I would use `require.Len` first on the slice so I don't see panics in the output, which make the output harder to read).
- `assert` is a normal assertion, the test will continue and fail, and you will see in the output all of the ones that failed.

In this case this is the last thing in the test and its a cleanup, it doesn't make sense that we `panic` / stop everything else in the test just because the cleanup failed, marking the test as failed is enough.

Reply:
Did not know that. Thanks for explaining